README.md

```diff
@@ -80,16 +83,54 @@ If you want to see how other projects have integrated with CodePush, you can che
 
 ## iOS Setup
 
-Once you've acquired the CodePush plugin, you need to integrate it into the Xcode project of your React Native app. To do this, take the following steps:
+Once you've acquired the CodePush plugin, you need to integrate it into the Xcode project of your React Native app and configure it correctly. To do this, take the following steps:
 
 ### Plugin Installation (iOS)
 
+In order to accomodate as many developer preferences as possible, the CodePush plugin supports iOS installation via three mechanisms:
+
+1. [**RNPM**](#plugin-installation-ios---rnpm) - [React Native Package Manager (RNPM)](https://github.com/rnpm/rnpm) is an awesome tool that provides the simplest installation experience possible for React Native plugins. If you're already using it, or you want to use it, then we recommend this approach. 
+
+2. [**CocoaPods**](#plugin-installation-ios---cocoapods) - If you're building a native iOS app that is embedding React Native into it, or you simply prefer using [CocoaPods](https://cocoapods.org), then we recommend using the Podspec file that we ship as part of our plugin.
+
+3. [**"Manual"**](#plugin-installation-ios---manual) - If you don't want to depend on any additional tools or are fine with a few extra installation steps (it's a one-time thing), then go with this approach.
+
+#### Plugin Installation (iOS - RNPM)
+
+1. Run `rnpm link react-native-code-push`
```

Comment:
Should we include a step to `npm install -g rnpm` if they have not already done it? Just to be super idiot proof.